Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM buildpack-deps:stretch
+
+RUN \
+  set -e; \
+  apt update && apt install -y \
+    clang \
+    ninja-build; \
+  wget -qO- "https://cmake.org/files/v3.13/cmake-3.13.1-Linux-x86_64.tar.gz" \
+    | tar --strip-components=1 -xz -C /usr/local; \
```

Comment:
Similarly, what's the reason to download cmake instead of using `apt`? I'm not saying we should, just wanting to document the choice.

Reply:
I was under the impression that it wasn't possible to do that! 😅 I switched it over now.